csaf_files/IT/white/2024/va-24-201-01.json

```diff
@@ -94,7 +94,7 @@
             "branches": [
               {
                 "category": "product_version_range",
-                "name": "all versions",
+                "name": "<=4.8.1",
```

Comment:
https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#31231-branches-type---name-under-product-version

"all versions" is invalid. See documentation above.

Reply:
I agree with the above change.

Reply:
@amanion-cisa there is also this as an option:
Version Range Specifier (vers)
vers is an ongoing community effort to address the problem of version ranges. Its draft specification is available at [VERS].
vers MUST be used in its canonical form. To convey the term "all versions" the special string vers:all/* MUST be used.

Reply:
I used vers:all/* for the Adminer advisory, I think this is more correct informationally, see https://github.com/cisagov/CSAF/compare/develop...amanion-cisa:CSAF:jmfixes#diff-b68d7bbabc197e783ddff83a11b37d5c9e309aeec18c89cf5551b478586daac1L97-R166.